src/main/java/edu/kis/vh/nursery/FifoRhymer.java

```diff
@@ -0,0 +1,21 @@
+package edu.kis.vh.nursery;
+
+public class FifoRhymer extends DefaultCountingOutRhymer {
+
+	private DefaultCountingOutRhymer internalRhymer = new DefaultCountingOutRhymer();
+
+	@Override
+	public int countOut() {
+		while (!isEmpty())
+
+			internalRhymer.countIn(super.countOut());
+
+		int ret = internalRhymer.countOut();
+
+		while (!internalRhymer.isEmpty())
+
+			countIn(internalRhymer.countOut());
+
```

Comment:
formatting convention issues (problems with merging 'format' branch?)
you forked a wrong repo